polygon_api/api.py

```diff
@@ -693,6 +693,13 @@ class Request:
     """
 
     def __init__(self, config, method_name, args=None):
```

Comment:
How about keeping all the methods as is, but converting everything to bytes before issuing the request?

This way the conversion logic won't be spread through all the code.

```python
class Request:

    ...

    @staticmethod
    def _encoded_args(args):
        return [(_value_to_utf8_bytes(key), _value_to_utf8_bytes(value))
                for key, value in args]

    @staticmethod
    def _value_to_utf8_bytes(value):
        if isinstance(value, bytes):
            return value
        if isinstance(value, bytearray):
            return bytes(value)
        return str(value).encode("utf-8")

    ...

    def issue(self):
        """Issues request and returns parsed JSON response"""

        args = list(self.args)
        args.append(('apiKey', self.config.api_key))
        args.append(('time', str(int(time.time()))))
        args.append(('apiSig', self.get_api_signature(args, self.config.api_secret)))
        response = requests.post(
            self.config.api_url + self.method_name, files=_encoded_args(_args))
        return Response(json.loads(response.text))

```

Reply:
There is an issue that API signature needs to be calculated using the arguments in their final look after the conversion, so I changed a little bit in new commits, please take a look.